sheetstorm/management/commands/generate_test_data.py

```diff
@@ -1,16 +1,27 @@
 import logging
+from enum import Enum
 
+from dateutil.relativedelta import relativedelta
 from django.contrib.auth import get_user_model
 from django.core.management.base import BaseCommand
 from django.db import transaction
+from django.utils import timezone
 
+from managers.factories import ProjectFactory
+from managers.models import Project
 from users.factories import UserFactory
 
 CustomUser = get_user_model()
 
 superuser_user_type = "SUPERUSER"
 
 
+class ProjectType(Enum):
```

Comment:
I was thinking about this and I think that we can leave it here as it is ;)

Reply:
Ok :)